src/App.tsx

```diff
@@ -15,36 +16,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ALPHABET = 'ALPHABET',
+  LENGTH = 'LENGTH',
+  '' = '',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: SortType,
+  reversed: boolean,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortType.ALPHABET:
+          return good1.localeCompare(good2);
+        case SortType.LENGTH:
+          return good1.length - good2.length;
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType['']);
+  const [reversed, setReversed] = useState<boolean>(false);
+  const visibleGoods = getPreparedGoods(goodsFromServer, sortField, reversed);
+  const isActive = (field: SortType): boolean => field !== sortField;
+
+  const reset = () => {
+    setSortField(SortType['']);
```

Comment:
The reset function sets `sortField` to `SortType['']`, which corresponds to the empty string in the enum. Update this to use a more meaningful default value after modifying the enum.